homepage.css

```diff
@@ -124,23 +123,75 @@ ul li:hover ul{
 	color:black;
 }
 
-.content_2{
-	background-color: white;
+
+.content_4{
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	z-index: 2;	
-	height: 150px;
-	text-align:center;
+	background-color: white;
```

Comment:
Rearrange for this div box because it hold the img and link to some small query.